bgpd/bgpd.h

```diff
@@ -681,6 +681,10 @@ struct bgp {
 	/* Weighted ECMP related config. */
 	enum bgp_link_bw_handling lb_handling;
 
+	/* BGP Conditional advertisement */
+	int condition_filter_count;
+	struct thread *t_condition_check;
```

Comment:
Shouldn't be this per AFI/SAFI?

Reply:
No Only one timer thread for polling. 
timer thread will be created/started when advertise-map is created on first neighbor
and will be canceled when the advertise-map is removed on the last neighbor.